src/modules/todoCRUD.js

```diff
@@ -0,0 +1,104 @@
+import Todo from './Todo-object.js';
+import { saveData, loadData } from './localStorage.js';
+
+class TodoList {
+  constructor() {
+    this.taskList = loadData();
+  }
+
+   // Add new item to To-Do List
+   addItem = (description) => {
+     this.taskList.push(new Todo(description, this.taskList.length));
+     saveData(this.taskList);
+     this.display();
+     document.getElementById('add-task').value = '';
+   };
+
+   // Delete todo item
+   deleteItem = (index) => {
+     this.taskList.splice(index, 1);
+     saveData(this.taskList);
+     this.display();
+   };
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that **Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.** But in the case of your For the example when I delete the 2nd item from the list the 3rd item is not updated to 2nd item in the local storage in the index value so Kindly Fix it. 
- To Do list in order(1-3 item)
![test 1](https://github.com/mulugithub/To-Do-List/assets/93598189/aa067c34-0f1c-480d-aa3a-69e8d20e1303)

- After deleting the second Item 

![test 1](https://github.com/mulugithub/To-Do-List/assets/93598189/ed4512f8-e8ff-4471-9286-e44de676f78f)

- After adding the3rd item in the index part it's duplicated.
![test 1](https://github.com/mulugithub/To-Do-List/assets/93598189/54d70bf3-7c62-44e0-a304-5123f446e9b3)